drivers/dai/intel/alh/alh.c

```diff
@@ -103,7 +112,7 @@ static int dai_alh_config_set(const struct device *dev, const struct dai_config
 	if (cfg->type == DAI_INTEL_ALH) {
 		return dai_alh_set_config_tplg(dp, bespoke_cfg);
 	} else {
-		return dai_alh_set_config_blob(dp, bespoke_cfg);
+		return dai_alh_set_config_blob(dp, cfg, bespoke_cfg);
```

Comment:
yeah I know this part is copy from other dais and we are kind of misusing the dai type. You could basically send the dai with a config for another dai type altogether. Maybe in the blob case we should check that the config type is DAI_INTEL_ALH_NHLT and otherwise return en error. But as said, we should do this in other dais also, so fine for this PR.